### ISSUE TITLE
support parallel test execution

### ISSUE THREAD

**Issue by aleksz** [beckje01/grails-code-coverage#32]
Would be nice to have configurable .ser file location and option to execute instrumentation before running tests in parallel.

**Comment by shendrix** [comment]
Agree - this is burning us right now running when multiple Jenkins jobs that use this plugin

**Comment by beckje01** [comment]
Config to move .ser file is now done as of #33
